internal/databases/postgres/wal_show_output_writer.go

```diff
@@ -52,7 +52,7 @@ func (writer *WalShowTableOutputWriter) Write(timelineInfos []*TimelineInfo) err
 	tableWriter.AppendHeader(header)
 
 	for _, tl := range timelineInfos {
-		row := table.Row{tl.ID, tl.ParentID, tl.SwitchPointLsn, tl.StartSegment,
+		row := table.Row{tl.ID, tl.ParentID, ConvertLSNtoPostgresFormat(tl.SwitchPointLsn), tl.StartSegment,
```

Comment:
Actually, you can utilize the `pgx.FormatLSN` function that does exactly this job.